client/index.html

```diff
@@ -1,11 +1,116 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+</head>
+
+<body>
+  <article class="Article Article--left">
+    <section class="Introduction">
+      <div class="Introduction__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
```

Comment:
W3C validator says error

![image](https://user-images.githubusercontent.com/22644149/182141246-e6343aa6-9037-466e-8392-5e99db99a23b.png)

Alt is required
https://html.spec.whatwg.org/multipage/images.html#alt

Reply:
Let’s use here `alt=""`.

`alt=""` is equal to `role="presentation"`

> Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it's decoration or a tracking pixel), and that non-visual browsers may omit it from rendering.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt